### PR TITLE
welcome: Add a simple welcome window when connecting to servers.

### PR DESCRIPTION
**Pull request by DolceTriade**
This can be overridden by server owners to add their own welcome window.

### CODE REVIEW

File: ui/welcome.rml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		<style>
+		panel {
+			width: 65%;
+			padding: 0.5em;
+		}
+		</style>
+	</head>
+	<body template="window" id="welcome" style="width:45em; margin: 1em;">
+		<h1>Welcome to Unvanquished</h1>
+		<tabset>
```
Comment (slipher) [inline ui/welcome.rml:13]: Why is there a tabset?

File: ui/welcome.rml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			<tab>
+				<img class="emoticon" src="/emoticons/official" />
+				<translate>Welcome</translate>
+			</tab>
+			<panel>
+				<translate>Welcome to Unvanquished</translate>
+				<br />
+				<br />
+				<translate>
```
Comment (slipher) [inline ui/welcome.rml:22]: We want to avoid leading/trailing spaces in translate tags

File: ui/welcome.rml (hunk 1 above)
Comment (DolceTriade) [reply ui/welcome.rml:13]: To allow people to add different tabs and such to their own welcome menu.

File: ui/welcome.rml (hunk 2 above)
Comment (DolceTriade) [reply ui/welcome.rml:22]: done.

**Comment by bmorel** [comment]
lgtm

**Comment by necessarily-equal** [comment]
@DolceTriade You need to merge this and update the submodules. In the meantime, people get this error on master and unv refuses to start:

```
Warn: Unable to open file }. 
Warn: Error during initialization: CGame VM: Error parsing ui/rocket.txt. Expecting "}" but found "{".
[Detaching after fork from child process 316697]
```

**Comment by DolceTriade** [comment]
I can add a change to make that welcome thing optional instead?

**Comment by slipher** [comment]
> I can add a change to make that welcome thing optional instead?

Sure, whatever. As long as it doesn't abort on startup.

**Comment by slipher** [comment]
I reverted it (in both repos) for the moment.

**Pull request merged**